src/document_array_embeddings.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import pytest
+from jina import Document, DocumentArray
+from jina.types.arrays.memmap import DocumentArrayMemmap
+
+from .utils.benchmark import benchmark_time
+
+NUM_REPETITIONS = 10
+
+# IMPORTANT: This benchmark currently is covered by
+# - document_array_save_binary_load_binary.py
+# - document_array_save_json_load_json.py
+# Only relevant if for future releases `.save` expands to other methods
```

Comment:
Is this part relevant to this file?

Reply:
We can remove it, but probably will forget about it then